packages/manager/modules/iam/src/resolves/misc.resolve.js

```diff
@@ -79,6 +79,34 @@ statementResolve.resolves = [entityResolve];
 
 // ---------------------------------------------------------------------------------------------------- //
 
+/**
+ * Get the status of the advanced mode
+ * @returns {boolean}
+ */
+const advancedModeResolve = /* @ngInject */ ($q, PreferencesService) => {
+  return $q
+    .resolve(PreferencesService.getPreference(KEY_PREFERENCES.advancedMode))
+    .then(({ value }) => {
+      return value === 'true';
```

Comment:
**question:** Are you sure you want to use `true` as a string instead of boolean value?

Reply:
yes, the api /me/preferences/manager return a string / text value.